projects/core/src/cart/facade/cart-data.service.ts

```diff
@@ -61,6 +61,14 @@ export class CartDataService {
   }
 
   private isEmail(str: string): boolean {
-    return /^[^\s@]+@[^\s@]+\.[^\s@]+$/.test(str);
+    if (str) {
+      return str.match(
+        // Email Standard RFC 5322:
+        /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/ // tslint:disable-line
```

Comment:
I tried to think of an other way but I could find one. Maybe this should be in a constant for both places?